src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -123,7 +127,9 @@ public InMoov(String n) {
   transient public OpenCV opencv;
   public Vision vision;
   transient public SpeechRecognizer ear;
+  public SpeechRecognition speechRecognition;
   transient public SpeechSynthesis mouth;
+  public SpeechSynthesizer speechSynthesizer;
```

Comment:
please remove this class.  it's unnecessary and should not be here.  If there is missing functionality from the speech synthesis interface class, we should add the appropriate functionality to be generic.  not specific to inmoov.

Reply:
That is not unnecessary, that is for SwingUI and and error management, but will revert to return later.. Like you said maybe by mods speech synthesis interface